### ISSUE TITLE
analyze() not working

### ISSUE THREAD

**Issue by b1azk0** [neuropsychology/psycho.R#87]
Whenever I call the `analyze()` funtion on a `lmerTest` object I get:

`Error in UseMethod("analyze") : 
  no applicable method for 'analyze' applied to an object of class "c('merModLmerTest', 'lmerMod', 'merMod')"`

```r
id = 15
obs = 10
data<- tibble::tibble(id = rep(1:id, obs), 
                      obs = rep(1:obs, 15), 
                      y = rnorm(150),
                      x = rnorm(150))

fit<-lmer(y ~ x + (1|id), data=data)
analyze(fit)
```
Converting `data` to `data.frame` does not help.

**Comment by DominiqueMakowski** [comment]
@b1azk0 This is likely due to the update of lmerTest and lme4, try updating `psycho`:
```
install.packages("devtools")
library("devtools")
install_github("neuropsychology/psycho.R")
library("psycho")
```
Let me know how it went

**Comment by b1azk0** [comment]
Unfortunatly it stops at:

`    values                                  html  
** building package indices
** installing vignettes
** testing if installed package can be loaded
*** arch - i386
Error: package or namespace load failed for 'psycho' in library.dynam(lib, package, package.lib):
 DLL 'statnet.common' not found: maybe not installed for this architecture?`

**Comment by b1azk0** [comment]
After manually reinstalling `statnet.common` it works again.
Thanks :)

**Comment by djhernanv** [comment]
I wanted to share that I had the same issue and could successfully get it to run after going trough both recommendations from @DominiqueMakowski  and @b1azk0

**Comment by ds-jim** [comment]
Coming to the package late in years, but I'm getting the peculiar error that `Error: 'analyze' is not an exported object from 'namespace:psycho'` . The function is still on the GitHub main page and is essential to the package yet doesn't seem to exist, even in the help `help(package = "psycho")`

**Comment by DominiqueMakowski** [comment]
Please check this post https://github.com/neuropsychology/psycho.R/issues/115#issuecomment-582221477

tldr; `analyze()` has now been improved and moved to the [**report**](https://github.com/easystats/report) package :)

Sorry for the trouble!

**Comment by ds-jim** [comment]
Thank you :-) guess I hadn't looked around long enough for the answer!
